src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    private static Random random = new Random();
```

Comment:
```suggestion
    private Random random = new Random();
```

Why static?

Reply:
I thought it was needed so that I am able to call method without instantiating an object. Now I see that I never actually used this feature and used the instance instead